packages/react-native/scripts/react_native_pods.rb

```diff
@@ -87,6 +89,7 @@ def use_react_native! (
 
   ENV['RCT_FABRIC_ENABLED'] = fabric_enabled ? "1" : "0"
   ENV['USE_HERMES'] = hermes_enabled ? "1" : "0"
+  ENV['AGGREGATE_PRIVACY_FILES'] = privacy_file_aggregation_enabled ? "1" : "0"
```

Comment:
```suggestion
  ENV['RCT_AGGREGATE_PRIVACY_FILES'] = privacy_file_aggregation_enabled ? "1" : "0"
```

We had issue in the past of flags clashing in some configurations, so better to prefix the React Native flags with RCT